content/agent/faq/agent-commands.md

```diff
@@ -66,8 +67,8 @@ The `[OK]` in the Agent output implies that the check was configured/run correct
 |Platform|Agent v5 |Agent v6|
 |:--------|:-----|:--------|
 |Linux|`sudo service datadog-agent info`|`sudo datadog-agent status`|
-|Docker|`sudo docker exec -it dd-agent /etc/init.d/datadog-agent info`|`n/a`|
+|Docker|`sudo docker exec -it dd-agent /etc/init.d/datadog-agent info`|`sudo docker exec -it datadog-agent agent status`|
```

Comment:
there's an extra `agent`

Reply:
datadog-agent is the container name, agent is the binary. I agree it's confusing tho, want me to change to `$AGENT_CONTAINER_NAME` or something?

Reply:
my bad, didn't realize `agent` was in the PATH, good for me and I imagine users will just copy/paste whatever we put in the docs so it might be better like this